source/basic.tex

```diff
@@ -550,6 +550,15 @@
 requirement applies recursively)\footnote{\ref{dcl.fct.default}
 describes how default argument names are looked up.}; and
 
+\item if \tcode{D} is a function that
+contains an assertion,
+has a postcondition,
+or invokes a function with a precondition,
+the conditions under which each definition of \tcode{D}
```

Comment:
This "the conditions under which" phrasing is awkward. Presumably the intent is that an implementation can say that some combinations are allowed and some are not, so we can't just say:

> [...] it is implementation-defined whether all definitions of `D` shall be translated using the same build level and violation continuation mode.

?

Reply:
Yes, it's my understanding that the implementation has to actually state the list of conditions, and the authors hope (in vain?) it's not "always", but something more modular and user-friendly.

Reply:
I've rephrased a bit more; it's now less awkward and still talks about "conditions".